stable/sysdig/values.yaml

```diff
@@ -111,6 +111,18 @@ secure:
   # true here enables Sysdig Secure: container run-time security & forensics
   enabled: false
 
+auditLog:
+  # true here activates the K8s Audit Log feature for Sysdig Secure
+  enabled: false
+  # Required: You need to specify a clusterIP for the service which listens to K8s audit events
+  clusterIP:
+  auditServerUrl: 0.0.0.0
+  auditServerPort: 7765
+
+  dynamicBackend:
+    # true here configures an AuditSink who will receive the K8s audit logs
+    enabled: false
```

Comment:
@nestorsalceda why don't we enable this one by default (once auditLog is enabled).

Reply:
That's because that is only one strategy to forward the logs to the Sysdig Agent and although it was rolled out on Kubernetes 1.13 it is still not the default configuration:

* For example in Kops the support is still landing: https://github.com/kubernetes/kops/blob/master/docs/cluster_spec.md#dynamic-audit-configuration
* In the EKS front, some falco users did a bridge between CloudWatch and the audit log server: https://xebia.com/blog/monitoring-aws-eks-audit-logs-with-falco/
* For GKE is quite simliar to EKS but with StackDriver logging: https://cloud.google.com/kubernetes-engine/docs/how-to/audit-logging

I prefer not to assume that Dinamic Backend is enabled in all platforms by default. I think that just deploying the service to ingest the logs will work better and allows the users to use their custom solutions.

Thanks for the review!